### ISSUE TITLE
NTR: Bone scan: positive (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9014]
New term request Bone scan: positive (seidel_190104161339): Technetium; abnormal  UMLS: 0740353

**Comment by pnrobinson** [comment]
@MickeySegal

this is probably too general to be a good HPO term. Could you provide context or specify?

**Comment by MickeySegal** [comment]
We use this for diseases such as Paget disease and bone and osteoid osteoma.  I've changed our finding name to "Bone scan (scintigraphy): abnormally increased uptake".

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001022
- label: Elevated bone scintigraphy uptake
- parents: [Abnormal skeletal morphology (HP:0011842)](https://hpo.jax.org/app/browse/term/HP:0011842)
- pmid: PMID:21889871
- def: Bone scintigraph is performed with technetium 99m-labeled diphosphonates. The uptake of these radiopharmaceuticals depicts osteoblastic activity and regional blood flow to bone.
- comment: Bone scintigraphy is frequently performed as part of the routine staging of patients with a wide variety of primary malignancies. Although it is a sensitive imaging modality, it is relatively nonspecific, and there are many potential causes of increased isotope uptake that can mimic metastatic disease.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9014
- synonyms: Increase bone scan uptake(NONE)|Bone scan (scintigraphy): abnormally increased uptake(NONE)
